ivadomed/config/config_small.json

```diff
@@ -4,7 +4,7 @@
     "log_directory": "spineGeneric_small",
     "debugging": false,
     "loader_parameters": {
-        "bids_path": "../duke/projects/ivado-medical-imaging/spineGeneric_201907041011/result/",
+        "bids_path": "../duke/projects/ivadomed/spineGeneric_201907041011/result/",
```

Comment:
Is the folder renamed on duke?

Reply:
good point! i was planning to but forgot 😉  will do soon (after making sure it is not mounted on everyone's end)